ICT4Dat/ict4dat-news-android
# #277 - Update Gradle and Dependencies, Prepare Release

- Update Gradle to latest version
- Update all Dependencies
- Adapt to new Ktlint formatting and add exception rules, you can run the Gradle Task `ktlintcheck` to see the results -> should succeed
- Tested Sentry
- Update App version to 1.4.0 to prepare next release
